astropy/utils/tests/test_data.py

```diff
@@ -2138,10 +2138,13 @@ def test_clear_download_cache_variants(temp_cache, valid_urls):
     assert not is_url_in_cache(u)
 
 
-@pytest.mark.skipif("CI", reason="Flaky on CI")
+@pytest.mark.skipif(CI and os.environ.get("IS_CRON", "false") == "false",
```

Comment:
Checking for CI would skip even the cron job, right? It is still technically CI because it is run in Actions.

```suggestion
@pytest.mark.skipif(os.environ.get("IS_CRON", "false") == "false",
```

Reply:
Thus the `and` – we don't want to disable it for everything else, do we (would never ever have found it in the RC2-tests then...)?

Reply:
Hmm I guess we can leave it in... maybe someone could have defined their own `IS_CRON` on their local machine for whatever reason.

Reply:
Which would not matter unless they also made a habit of defining `CI` locally... 🤔